common-requests/use.md

```diff
@@ -1,8 +1,176 @@
+
+{% capture code %}hzn agreement list{% endcapture %}
+{% include code_snippet.md code=code language='shell' %}
+
+This command displays a JSON array of objects showing one or more active agreements between the edge node and AgBots:
+
+``` json
+[
+  {
+    "name": "Policy for myorg/node1 merged with myorg/policy-ibm.helloworld_1.0.0",
+    "current_agreement_id": "83785680de5d22c874a0f54dc229738a0e744db22b2c0deeb320b9fdf0967138",
+    "consumer_id": "IBM/agbot",
+    "agreement_creation_time": "2020-08-04 16:44:36 -0400 EDT",
+    "agreement_accepted_time": "",
+    "agreement_finalized_time": "",
+    "agreement_execution_start_time": "",
+    "agreement_data_received_time": "",
+    "agreement_protocol": "Basic",
+    "workload_to_run": {
+      "url": "ibm.helloworld",
+      "org": "IBM",
+      "version": "1.0.0",
+      "arch": "amd64"
+    }
+  }
+]
+```
+
+Note the following properties:
+
+| Property | Description |
+| --- | --- |
+| `0.agreement_creation_time` | When the agreement was proposed by the AgBot |
+| `0.agreement_accepted_time` | When the agreement was accepted by the edge node |
+| `0.agreement_finalized_time` | When the agreement was completed |
+| `0.agreement_execution_start_time` | When the edge node began downloading the container image(s) |
+
+After you see the `agreement_execution_start_time` value, you can run `docker ps` to confirm that the workloads have started.
+
+## More activities
+
+| Task | Command |
+| --- | --- |
+| View the default "helloworld" sample service | `hzn service log -f ibm.helloworld` |
+| View the agreement negotiation logged steps | `hzn eventlog list` |
+| View the node policy | `hzn policy list` |
+
+## Communicate with the exchange
+
+To use the exchange communication commands, set up two environment variables.
+
+### Pre-requisite
+
+You need this information from the all-in-one installation summary message displayed when you installed the management hub, agent, and CLI.  The summary message is similar to this example:
+
+``` text
+----------- Summary of what was done:
+  1. Started Horizon management hub services: agbot, exchange, postgres DB, CSS, mongo DB
+  2. Created exchange resources: system org (IBM) admin user, user org (myorg) and admin user, and agbot
+     - Exchange root user generated password: 1234567890
+     - System org admin user generated password: AbcDEfg
+     - Agbot generated token: Abc123YZ
+     - User org admin user generated password: XLmdsg236
+     - Node generated token: uingtw398J
+     Important: save these generated passwords/tokens in a safe place. You will not be able to query them from Horizon.
+  3. Installed the Horizon agent and CLI (hzn)
+  4. Created a Horizon developer key pair
+  5. Installed the Horizon examples
+  6. Created and registered an edge node to run the helloworld example edge service
+For what to do next, see: https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#all-in-1-what-next
```

Comment:
Missing closing period in sentence. And general question...in my *.md files, the cross-reference's are formatted differently, but i am not sure it that matters in this format.